server.js

```diff
@@ -1,13 +1,217 @@
 const express = require('express');
 const app = express();
 
-app.use(express.json());
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('./knexfile')[environment];
+const database = require('knex')(configuration);
 
+app.use(express.json());
 
 app.set('port', process.env.PORT || 3000);
 
 app.get('/', (request, response) => {
-  response.send('Oh hey Pet Box');
+  response.send('Reached Palette Picker');
+});
+
+app.get('/api/v1/projects', async (request, response) => {
+  try {
+    const projects = await database('projects').select();
+    return response.status(200).json(projects);
+  } catch (error) {
+    return response.status(500).json({ error });
+  }
+});
+
+app.get('/api/v1/projects/:id', async (request, response) => {
+  const { id } = request.params;
+
+  if(!parseInt(id)) {
```

Comment:
This is good error handling to make sure that the id put in is an integer and if anything else it tells them that the id has to be an integer. Nice work!